### ISSUE TITLE
Subheadings in the navigation menu don't work

### ISSUE THREAD

**Issue by benesva4** [doczjs/docz#1577]
Hi, the navigation menu on the left side of generated docs works only on the H1 level. It doesn't scroll when you click on the subheading even though url changes. On the other hand if you click on the subheading itself in the docs (the main part) it works. When you scroll, subheadings are not getting marked as active, but when you click a subheading and then reload, that subheading gets active styling.

**Comment by aymeric-giraudet** [comment]
Yup, I second that, here's an example here : https://aymeric-giraudet.github.io/tsuurukitto/src-components-alert-alert#properties

**Comment by secreter** [comment]
mark,me too.
